website/src/components/header/home-page-header.js

```diff
@@ -187,14 +187,12 @@ const HeroIntro = () => {
 				<Cell width={[10, 12, 10, 10]} left={[2, 1, 2, 2]}>
 					<Heading
 						size={1}
+						fontType="brandFont"
 						css={mq({
 							paddingTop: [SPACING(4), SPACING(7)],
 							paddingBottom: [SPACING(4), SPACING(6)],
-							fontFamily: `${TYPE.brandFont.fontFamily} !important`,
 							maxWidth: 600,
-							fontWeight: 500,
 							margin: '0 auto !important',
-							zIndex: 3,
```

Comment:
Why would the z-index change?

Reply:
Just not sure why it's there... any idea?

Reply:
I'd assume cause of the header?